BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1211,3 +1210,4 @@
 "LoggingEndsOnDateAtTime" = "Logging ends on %1$@ at %2$@";
 "YouMustAttachAFileToSaveThisSend" = "You must attach a file to save this send";
 "SendUpdated" = "Send updated";
+"WeDontRecognizeThisDeviceEnterTheCodeSentToYourEmailDescriptionLong" = "We don't recognize this device. Enter the code sent to your email to verify your identity.";
```

Comment:
⛏️ I see that aside from the number of digits, the email was also removed from the message. Is that ok?

Reply:
Yes that was intended